packages/big-design-theme/src/stitches.ts

```diff
@@ -13,7 +14,7 @@ export const { styled, config, createTheme, css, getCssText, globalCss, keyframe
     theme: {
       colors: { ...colors },
       space: { ...createSpacing() },
-      sizes: { ...createSpacing() },
+      sizes: { ...createSpacing(), ...breakpointValues },
```

Comment:
Note: I added the breakpointValues in the `sizes` section located in the  stitches `theme` file in order to get access to these values. Is that ok?